src/sentry/static/sentry/app/utils/advancedAnalytics.tsx

```diff
@@ -0,0 +1,89 @@
+import * as qs from 'query-string';
+import {IntegrationsEventParams} from 'src/sentry/static/sentry/app/utils/integrationUtil';
+
+import {Organization} from 'app/types';
+import {trackAnalyticsEvent} from 'app/utils/analytics';
+import {uniqueId} from 'app/utils/guid';
+
+const ANALYTICS_SESSION = 'ANALYTICS_SESSION' as const;
```

Comment:
I don't think you need the `as const` here

Reply:
Yea, idk why that was there before